go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -188,30 +170,17 @@ func (ef *egressFirewall) addACLToJoinSwitch(joinSwitches []string, hashedAddres
 		if len(rule.ports) > 0 {
 			match = fmt.Sprintf("%s && ( %s )\"", match[:len(match)-1], egressGetL4Match(rule.ports))
 		}
-		uuid, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading",
-			"--columns=_uuid", "find", "ACL", match, "action="+action,
-			fmt.Sprintf("external-ids:egressFirewall=%s", ef.namespace))
 
+		_, stderr, err := util.RunOVNNbctl("--id=@logical_router_policy", "create", "logical_router_policy",
+			fmt.Sprintf("priority=%d", defaultStartPriority-rule.id),
```

Comment:
@JacobTanenbaum now that our range is smaller, do we have a problem with number of rules? Or do we cap at 99 or something?

Reply:
so we can cap the supported number of rules to 99 but that is also the number of rules in a namespace, not for the system as a whole and currently there is a limit in openshift sdn of 50 so it is higher then the sdn version. Lastly if we ever run out of space we can raise all the other priorities to give the firewall more room at the base.

Reply:
Note that Dan Winship filed a PR for SDN to increase that number to 1000 or something, so this is no longer a good rationale.